Sources/CheckResolved.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Sources/CheckResolved.swift
+//  SwinjectAutoregistration
+//
+//  Generated by Swinject AutoRegistration generator.
+//  Copyright © 2017 Swinject Contributors. All rights reserved.
+//
+
+
+ import Swinject 
+
+
+func unresolvedService<A>(_ a: A?) -> String? {
+   return ( a == nil ? "\(A.self)" : nil )
+}
+
+func checkResolved<Service, A>(initializer: (A) -> Service, services a: A?){
+   let unresolved = ( [a] as [Any?] ).filter { $0 == nil }
+   if unresolved.count > 0 {
+       let warningsMessage = warnings(forInitializer: initializer).map { "\($0.message)\n" }.joined()
+       fatalError("SwinjectAutoregistration: Resolution failed.\n\(warningsMessage)Unresolved service: \(unresolvedService(a)!)\nInitializer: (\(A.self)) -> \(Service.self)")
```

Comment:
If this is just a warning message, shouldn't this be `print` or `NSLog`? The other warnings in this file are the same.